crates/emitter/src/emitter.rs

```diff
@@ -333,6 +339,7 @@ impl Emitter {
                 }
             }
             dbg!("a");
+            dbg!(arg);
```

Comment:
Sorry. The above `dbg` is mistake.
Could you remove all `dbg`?

Reply:
My mistake. Fixed.